### PR TITLE
Clarify how to configure Ruler to send alerts to Alertmanager

### PR DESCRIPTION
**Pull request by charleskorn**
#### What this PR does

It took me a bit of trial and error to figure out how to configure Ruler to send alerts to Alertmanager - this updates the docs to reflect how I _think_ we want it to be configured.

#### Which issue(s) this PR fixes or relates to

Relates to #3684 

#### Checklist

- [n/a] Tests updated
- [n/a] Documentation added
- [n/a] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by pracucci — APPROVED** [review]
LGTM! (modulo a nit)

@osg could you take a look, please?

File: docs/sources/mimir/operators-guide/architecture/components/alertmanager.md [hunk 1]
```diff
@@ -117,3 +117,11 @@ The Mimir Alertmanager also periodically stores the alert state in the storage b
 When an Alertmanager starts, it attempts to load the alerts state for a given tenant from other Alertmanager replicas. If the load from other Alertmanager replicas fails, the Alertmanager falls back to the state that is periodically stored in the storage backend.
 
 In the event of a cluster outage, this fallback mechanism recovers the backup of the previous state. Because backups are taken periodically, this fallback mechanism does not guarantee that the lastest state is restored.
+
+## Ruler configuration
+
+[Ruler]({{< relref "ruler/index.md" >}}) must be configured with the addresses of Alertmanagers with the `-ruler.alertmanager-url` flag.
```
Comment (osg-grafana) [inline docs/sources/mimir/operators-guide/architecture/components/alertmanager.md:123]:
```suggestion
You must configure the [ruler]({{< relref "ruler/index.md" >}}) with the addresses of Alertmanagers via the `-ruler.alertmanager-url` flag.
```

Comment (osg-grafana) [reply docs/sources/mimir/operators-guide/architecture/components/alertmanager.md:123]: Rationale: Use active voice where possible. Also, I changed "with" to "via" here to read more smoothly; this change is a word choice change.

File: docs/sources/mimir/operators-guide/architecture/components/alertmanager.md [hunk 2]
```diff
@@ -117,3 +117,11 @@ The Mimir Alertmanager also periodically stores the alert state in the storage b
...
+
+The address should point to Alertmanager's API.
```
Comment (osg-grafana) [inline docs/sources/mimir/operators-guide/architecture/components/alertmanager.md:125]:
```suggestion
Point the address to Alertmanager’s API.
```
The term "should" is generally ambiguous; don’t be afraid to write directly to the reader.

File: docs/sources/mimir/operators-guide/architecture/components/alertmanager.md [hunk 3]
```diff
@@ -117,3 +117,11 @@ The Mimir Alertmanager also periodically stores the alert state in the storage b
...
+Alertmanager's API prefix is configured by the `-http.alertmanager-http-prefix` flag, and defaults to `/alertmanager`.
```
Comment (osg-grafana) [inline docs/sources/mimir/operators-guide/architecture/components/alertmanager.md:126]:
```suggestion
You can configure the Alertmanager’s API prefix via the `-http.alertmanager-http-prefix` flag, which defaults to `/alertmanager`.
```

File: docs/sources/mimir/operators-guide/architecture/components/alertmanager.md [hunk 4]
```diff
@@ -117,3 +117,11 @@ The Mimir Alertmanager also periodically stores the alert state in the storage b
...
+For example, if Alertmanager is listening at `https://mimir-alertmanager.namespace.svc.cluster.local` and is using the default API prefix, set `-ruler.alertmanager-url` to `https://mimir-alertmanager.namespace.svc.cluster.local/alertmanager`.
```
Comment (osg-grafana) [inline docs/sources/mimir/operators-guide/architecture/components/alertmanager.md:127]:
```suggestion
For example, if Alertmanager is listening at `https://mimir-alertmanager.namespace.svc.cluster.local`, and it is using the default API prefix, set `-ruler.alertmanager-url` to `https://mimir-alertmanager.namespace.svc.cluster.local/alertmanager`.
```

File: docs/sources/mimir/operators-guide/architecture/components/ruler/index.md [hunk 5]
```diff
@@ -49,9 +49,13 @@ If an alerting rule has a defined `for` duration, it enters the **PENDING** (`pe
 After the alert has been active for the entire `for` duration, it enters the **FIRING** (`firing`) state.
 The ruler then notifies Alertmanagers of any **FIRING** (`firing`) alerts.
 
-Configure the addresses of Alertmanagers with the `-ruler.alertmanager-url` flag, which supports the DNS service discovery format.
+Configure the addresses of Alertmanagers with the `-ruler.alertmanager-url` flag. This flag supports the DNS service discovery format.
 For more information about DNS service discovery, refer to [Supported discovery modes]({{< relref "../../../configure/about-dns-service-discovery.md" >}}).
 
+If you're using [Mimir's Alertmanager]({{< relref "../alertmanager.md" >}}), the address should point to Alertmanager's API.
```
Comment (osg-grafana) [inline docs/sources/mimir/operators-guide/architecture/components/ruler/index.md:55]:
```suggestion
If you're using [Mimir's Alertmanager]({{< relref "../alertmanager.md" >}}), point the address to Alertmanager's API.
```

File: docs/sources/mimir/operators-guide/architecture/components/ruler/index.md [hunk 6]
```diff
@@ -49,9 +49,13 @@ If an alerting rule has a defined `for` duration, it enters the **PENDING** (`pe
...
+Alertmanager's API prefix is configured by the `-http.alertmanager-http-prefix` flag, and defaults to `/alertmanager`.
```
Comment (osg-grafana) [inline docs/sources/mimir/operators-guide/architecture/components/ruler/index.md:56]:
```suggestion
The Alertmanager's API prefix is configured via the `-http.alertmanager-http-prefix` flag, which defaults to `/alertmanager`.
```

**Pull request merged**